synthtool/gcp/templates/node_library/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
+- [ ] Make sure to open an issue as a bug/issue before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea.
```

Comment:
might be nice to link to `https://github.com/googleapis/[LIBRARY]/issues/new/choose`, using a template variable.